modules/vm/variables.tf

```diff
@@ -14,6 +16,8 @@ variable "resource_group_name" {
   type        = string
 }
 
+#
```

Comment:
comment ?

Reply:
Preparation for separating the different groups of configurations